doc/C_STYLE_GUIDE.rst.txt

```diff
@@ -166,7 +166,7 @@ Code lay-out
   Note that the terms in the multi-line Boolean expression are indented so
   as to make the beginning of the code block clearly visible.
 
-* Put blank lines around functions, structure definitions, and
+* Put blank lines around functions, structure definitions and
```

Comment:
[Oxford comma](https://blog.oxforddictionaries.com/2015/01/21/video-oxford-comma/) is fine, I'd leave this alone.